utils.py

```diff
@@ -0,0 +1,182 @@
+import os
+import re
+import BiLSTM_CRF
+from collections import Counter
+
+models_path = "./models"
+eval_path = "./evaluation"
+eval_temp = os.path.join(eval_path, "temp")
+eval_script = os.path.join(eval_path, "conlleval")
+
+
+def create_mapping(dico):
+
+    sorted_items = sorted(dico.items(), key=lambda x: (-x[1], x[0]))
+    id_to_item = {i: v[0] for i, v in enumerate(sorted_items)}
+    item_to_id = {v: k for k, v in id_to_item.items()}
+    return item_to_id, id_to_item
+
+
+def iob_to_iobes(tags):
+    new_tags = []
+    for i, tag in enumerate(tags):
+        if tag == "O":
+            new_tags.append(tag)
+        elif tag.split("-")[0] == "B":
+            if i + 1 != len(tags) and tags[i + 1].split("-")[0] == "I":
+                new_tags.append(tag)
+            else:
+                new_tags.append(tag.replace("B-", "S-"))
+        elif tag.split("-")[0] == "I":
+            if i + 1 < len(tags) and tags[i + 1].split("-")[0] == "I":
+                new_tags.append(tag)
+            else:
+                new_tags.append(tag.replace("I-", "E-"))
+        else:
+            raise Exception("Invalid IOB format!")
+    return new_tags
+
+
+def load_sentences(path, zeros):
+    sentences = []
+    sentence = []
+    for line in open(path, 'r', encoding='utf-8'):

```

Comment:
always use `with` for opening files. 
```suggestion
    with open(path, 'r', encoding='utf-8') as file:
        for line in file:    
```